src/game/client/components/menus_demo.cpp

```diff
@@ -151,12 +151,13 @@ void CMenus::RenderDemoPlayer(CUIRect MainView)
 	};
 
 	static int s_SkipDurationIndex = 1;
```

Comment:
The default should still be 1s
```suggestion
	static int s_SkipDurationIndex = 3;
```

Reply:
true